src/test/java/edu/ucla/library/iiif/auth/delegate/CantaloupeAuthDelegateIT.java

```diff
@@ -132,280 +174,474 @@
+
+            Assert.assertEquals(HTTP.UNAUTHORIZED, response.statusCode());
+            Assert.assertTrue(TestUtils.responseHasContentType(response, MediaType.APPLICATION_JSON,
+                    MediaType.APPLICATION_LD_PLUS_JSON));
+            TestUtils.assertEquals(expectedResponse, response.body());
+        }
     }
 
     /**
-     * Generates the expected info.json for a given image.
-     *
-     * @param aImageID The identifier of the image whose info we're requesting
-     * @param aResponseTemplate The response template that we should use to render the response
-     * @param aImageApiVersion The IIIF Image API endpoint to use
-     * @return The expected info.json response for the image
-     * @throws IOException If there is trouble reading the test file
+     * Tests for image requests via Image API 2.
      */
-    private static String getExpectedDescriptionResource(final String aImageID, final File aResponseTemplate,
-            final int aImageApiVersion)
-                    throws IOException {
-        final Map<String, String> envProperties = System.getenv();
-        final String descriptionResourceID =
-                getDescriptionResourceID(envProperties.get(TestConfig.IIIF_URL_PROPERTY), aImageApiVersion, aImageID);
-        final List<String> responseTemplateURLs = new ArrayList<>();
+    public static class ImageRequestV2IT extends AbstractRequestIT {
 
-        responseTemplateURLs.add(descriptionResourceID);
+        @Override
+        public void testFullAccessResponseOpenUnauthorized() throws IOException, InterruptedException {
+            final HttpResponse<byte[]> response = sendImageRequest(OPEN_ACCESS_IMAGE, null, 2);
+            final byte[] expectedResponse = getExpectedImage(OPEN_ACCESS_IMAGE);
 
-        if (aResponseTemplate.equals(DEGRADED_ACCESS_RESPONSE_TEMPLATE_V2) ||
-                aResponseTemplate.equals(DEGRADED_ACCESS_RESPONSE_TEMPLATE_V3)) {
-            // The Hauth service URLs need to be added to the info.json
-            responseTemplateURLs.add(envProperties.get(Config.AUTH_COOKIE_SERVICE));
-            responseTemplateURLs.add(envProperties.get(Config.AUTH_TOKEN_SERVICE));
-        } else if (aResponseTemplate.equals(NO_ACCESS_RESPONSE_TEMPLATE_V2) ||
-                aResponseTemplate.equals(NO_ACCESS_RESPONSE_TEMPLATE_V3)) {
-            responseTemplateURLs.add(envProperties.get(Config.SINAI_AUTH_TOKEN_SERVICE));
+            Assert.assertEquals(HTTP.OK, response.statusCode());
+            Assert.assertTrue(TestUtils.responseHasContentType(response, MediaType.IMAGE_TIFF));
+            Assert.assertTrue(Arrays.equals(expectedResponse, response.body()));
         }
 
-        return StringUtils.format(StringUtils.read(aResponseTemplate), responseTemplateURLs.toArray());
+        @Override
+        @Test
+        @Ignore
+        public void testFullAccessResponseTieredAuthorized() throws IOException, InterruptedException {
+            // TODO Auto-generated method stub
```

Comment:
Is there another ticket for this already?

Reply:
IIIF-1128 😉 (I think)

Reply:
I've added a note there to make sure they're done with that ticket. :-)

Reply:
Oops, I meant IIIF-1125